### DISCUSSION TITLE
Enhancement - Better Hyperparameters

### DISCUSSION THREAD

**Discussion by Praful932** [Praful932/Kitabe#27] [Ideas]
Since we earlier used Surprise to select the best hyperparameters and generate the Embedding, and we were earlier limited by compute power.
Maybe we can use [tf-rec](https://github.com/Praful932/tf-rec) now to search for better params

What do you think @maneprajakta

**Comment by maneprajakta** [comment]
Yes we can :smile: @Praful932
